buildQuery.R

```diff
@@ -0,0 +1,18 @@
+library(readr)
+library(SkeletonCohortCharacterization)
+library(DatabaseConnector)
+
+design <- read_file('cc.json')
+
+connectionDetails <- DatabaseConnector::createConnectionDetails(dbms = "postgresql",
+      connectionString = "jdbc:postgresql://odysseusovh02.odysseusinc.com/cdm_v500_synpuf_v101_110k",
+      user = "ohdsi",
+      schema = "public",
+      password = "ohdsi")
+
+#runAnalysis(connectionDetails, design, cohortTable='cohort', sessionId='abc',  cdmSchema='public', resultsSchema='results', vocabularySchema='public', analysisId=3334)
+
+#q <- buildQuery(design, cohortTable='cohort', sessionId='',  cdmSchema='public', resultsSchema='results', vocabularySchema='public', analysisId=3)
+#print(paste('Query: ', q))
```

Comment:
What is this file for? Why does it contain commented out lines? And especially why it contains credentials inside?